src/lib/data.ts

```diff
@@ -69,12 +69,13 @@ export const speakers: Speaker[] = [
       "Matas Strukčinskas is part of the Devpod team at Uber. <a href='https://www.uber.com/en-DE/blog/devpod-improving-developer-productivity-at-uber/' target='_blank'>Read about Uber’s CDE approach.</a>",
   },
   {
-    image: "/images/speakers/pauline.webp",
-    name: "Pauline Narvas",
-    title: "Head of Community at Gitpod",
-    twitter: "paulienuh",
-    linkedin: "https://www.linkedin.com/in/pnarvas",
-    imgTitle: "MC for CDE Universe",
+    image: "/images/speakers/shelby.webp",
+    name: "Shelby Carpenter",
+    title: "Senior Product Marketing Manager at MongoDB",
+    twitter: "shelbscarps",
+    github: "getsendy",
+    imgTitle:
+      "Shelby will give a glimpse of the role of a cloud database within cloud-native development workflows.",
```

Comment:
@pawlean Is it fine, or do you have some other suggestions?

Reply:
@Siddhant-K-code Can it be this instead:

> Shelby will share the role of cloud database within cloud-native development workflows.